doc/src/sgml/installation-meson.sgml

```diff
@@ -0,0 +1,2635 @@
+<!-- doc/src/sgml/installation-meson.sgml -->
+<!--
+
+The standalone version has some portions that are different from the version
+that is integrated into the full documentation set, in particular as regards
+links, so that INSTALL.html can be created without links to the main
+documentation.  See standalone-profile.xsl for details.
+
+-->
+
+<chapter id="installation-meson">
+ <title>Installation from Source Code with Meson</title>
+
+ <indexterm zone="installation">
+  <primary>installation</primary>
+ </indexterm>
+
+ <!-- See also the version of this text in standalone-install.xml -->
+ <para>
+  This chapter describes the installation of
+  <productname>PostgreSQL</productname> using the source code
+  distribution.  If you are installing a pre-packaged distribution,
+  such as an RPM or Debian package, ignore this chapter
+  and see <xref linkend="install-binaries" /> instead.
+ </para>
+
+ <para>
+  What is Meson and Why Meson?
```

Comment:
I'm not sure that makes a lot of sense to have here?

Reply:
Removed. But, I think we should have some content on why users should use one over the other OR why we recommend one over the other.